library/globals.inc.php

```diff
@@ -1144,6 +1144,19 @@
       xl('Total Minimum Dollar Amount of Statement to Allow Printing.(only applicable if Allow Statement to be Excluded from Printing is enabled)')
     ),
 
+    'insurance_statement_exclude' => array(
+       xl('Do Not Print Statements For Insurance Companies'),
+       array(
+            '0' => xl('Primary'),
+            '1' => xl('Secondary'),
+            '2' => xl('Tertiary'),
```

Comment:
Multi-select?  Not looking at the test (doing that tomorrow) but does the logic work out with the default that is only prints out the statement for Primary?

Reply:
No Multi select. If the primary is selected then no statement will be printed if that insurance is primary.